### PR TITLE
Rename bind_hostname to bind_address

### PR DESCRIPTION
**Pull request by gsnedders**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/9998)

### CODE REVIEW

File: config.default.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
           "wss":["auto"]},
  "check_subdomains": true,
  "log_level":"debug",
- "bind_hostname": true,
+ "bind_address": true,
```
Comment (gsnedders) [inline config.default.json]: Do we need to change anything so old `config.json` files still work? (These are only supported with `wpt serve` and not `wpt run`, AFAIK.)

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-13 15:21:00
Finished: 2018-03-13 15:37:49





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25860)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/352891403)

**Comment by jgraham** [comment]
Review status: 0 of 8 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[config.default.json, line 11 at r1](https://reviewable.io:443/reviews/w3c/web-platform-tests/9998#-L7UwhSyi49L18Ja5K7L-r1-11:-L7VD0RJfkvDVgDq05ZH:buntk3k) ([raw file](https://github.com/w3c/web-platform-tests/blob/92a14a0e80be45701cd430fe7219be5e043580d1/config.default.json#L11)):*
<details><summary><i>Previously, gsnedders (Geoffrey Sneddon) wrote…</i></summary><blockquote>

Do we need to change anything so old `config.json` files still work? (These are only supported with `wpt serve` and not `wpt run`, AFAIK.)
</blockquote></details>

Yes, I think explicit support for the old value with a warningn would be a good idea.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/web-platform-tests/9998)*

**Comment by gsnedders** [comment]
Review status: 0 of 8 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[config.default.json, line 11 at r1](https://reviewable.io:443/reviews/w3c/web-platform-tests/9998#-L7UwhSyi49L18Ja5K7L-r1-11:-L7VK5MBJ9A89pvJtkLD:b-896fix) ([raw file](https://github.com/w3c/web-platform-tests/blob/92a14a0e80be45701cd430fe7219be5e043580d1/config.default.json#L11)):*
<details><summary><i>Previously, jgraham wrote…</i></summary><blockquote>

Yes, I think explicit support for the old value with a warningn would be a good idea.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/w3c/web-platform-tests/9998)*

**Review by jgraham — APPROVED** [review]

**Pull request merged**